src/component/parser/json/converter/DiagramConverter.ts

```diff
@@ -80,7 +80,7 @@ export default class DiagramConverter extends AbstractConverter<BpmnModel> {
       }
 
       // TODO error management
-      console.log('Shape json deserialization: unable to find bpmn element with id %s', shape.bpmnElement);
+      console.error('Shape json deserialization: unable to find bpmn element with id %s', shape.bpmnElement);
```

Comment:
:question: why not putting a warning instead? Here we are just ignoring the element and so it is not displayed
This would require to authorize warning logs